examples/templates/css/sidebar.css

```diff
@@ -114,10 +121,48 @@ body {
     background: none;
 }
 
+.container-fluid {
+    padding-right: 15px;
+    padding-left: 15px;
+    margin-right: auto;
+    margin-left: auto;
+    height: inherit;
+}
+.row {
+    margin-right: -15px;
+    margin-left: -15px;
+    height: inherit;
+}
+
+.btn {
```

Comment:
Just copied the bootstrap properties but you are right, removed not needed properties.